rakefile.rb

```diff
@@ -95,6 +101,9 @@
 desc "Execute default tasks"
 task :default => [ :vars, :gitlink, :unit, :integ, :spec, :pack ]
 
+desc "Execute default tasks plus .NET Core tasks"
+task :dotnet => [ :default, :unit_dotnet, :pack_dotnet]

```

Comment:
Sorry, I'd written "`_dotnet` packages". I meant "`_dotnet` targets". I think you understood that anyhow.